pages/contributors.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contributors
+layout: default
+has_children: false
+---
+
+# Meet the Team
+
+### Jann Arellano
+Jann responsible for the Django backend due to his full-stack development and data analytics experience.
+### Brian Buslon
+Brian is resposible for for the Svelte frontend (both its visual design and implementation) due to his graphic design and web development experience.
+### Keaton Clark
+Keaton is resposible for the messaging model and the research and development of each communication protocol.
+### Lloyd Gonzales
+Lloyd is responsible for the implementation of agents due to his experience in malware research and general cybersecurity knowledge.
+
+# Advisor
+
+### Shamik Sengupta
+Shamik is a professor at the University of Nevada, Reno
+
+# Instructors
+
+### Sara Davis
+
+### David Feil-Seifer
+
+### Devrin Lee
+
+### Vinh Lee
```

Comment:
misspelled

```suggestion
### Vinh Le
```

Reply:
also i don't believe Vinh is counted as an instructor anyways, I think this should be removed